_episodes/conclusions.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Conclusion of the basic tutorial"
+teaching: 10
+exercises: 0
+questions:
+- "What do I do now?"
+- "Where can I get help?"
+- "What if I find a bug?"
+- "Where can I find more information about ESMValtool?"
+- "How can I cite ESMValtool?"
+
+objectives:
+- "breathe - you're finished now!"
+- "Congratulations & Thanks!”
+- "Find out about the mini-tutorials, and what to do next."
+---
+
+## Congratulations!
+
+Congratulations on completing the ESMValTool tutorial! You should be now ready to go and start using ESMValTool independently. 
+
+The rest of this tutorial contains individual mini-tutorials to help work through a specific issue. 
+
+
+### What next?
+
+From here, there are lots of ways that you can continue to use ESMValTool.
+
+- You can start from the list of existing recipes and run one of those. [LINK to ep 8]
+- You can learn how to write your own diagnostics and recipes. [LINK to ep 9]
+- You can contribute your recipe and diagnostics back into ESMValTool. [Link to ep 10]
+- You can learn how to prepare observational datasets to be suitable for use by ESMValTool. [ Link to ep 11]
+
+
+### Where can I get more information on ESMValTool?
+
+Read the docs page: https://esmvaltool.readthedocs.io/
+
+Technical description paper: https://doi.org/10.5194/gmd-13-1179-2020 
+
+Source code (ESMValTool): https://github.com/ESMValGroup/ESMValTool
+
+Source code (ESMValCore ): https://github.com/ESMValGroup/ESMValCore 
+
+
+### Where can I get more help?
+
+There are lots of resources available for helping you use ESMValTool. 
+
+If you get stuck, a great starting point is to create a new issue. [Link to ep 10]
+
+
+
+### What if I find a bug? 
+
+If you find a bug, please report it back to the ESMValTool team. This will help us fix it so that you can continue working, but also it means that ESMValTool will be more stable for everyone else as well. 
+
+Please create a new issue using this page: https://github.com/ESMValGroup/ESMValTool/issues
```

Comment:
Maybe rephrase to "To report a bug, please create..." that it is clear that for bug-reporting you have to create an issue.

Reply:
Resolved.